### DISCUSSION TITLE
Polls data clone

### DISCUSSION THREAD

**Discussion by lajczik** [discord-jda/JDA#2671] [Feedback & Suggestions]
Instread of this method:
`MessagePoll poll = msg.getPoll();`
`action.setPoll(new MessagePollData(poll.getLayout(), poll.getQuestion(), poll.getAnswers(), Duration.ofDays(3), poll.isMultiAnswer()));`

Add method like in MessageCreateData:
`action.setPoll(MessagePollData.fromPoll(msg.getPoll()));`

**Comment by MinnDevelopment** [comment]
How about `new MessagePollBuilder(msg.getPoll()).build()`?
